packages/clerk-js/src/ui/hooks/useEnabledThirdPartyProviders.tsx

```diff
@@ -37,25 +36,12 @@ const strategyToDisplayData: ThirdPartyStrategyToDataMap = fromEntries(
 
 export const useEnabledThirdPartyProviders = () => {
   const { socialProviderStrategies, web3FirstFactors, authenticatableSocialStrategies } = useEnvironment().userSettings;
-  const { experimental_enableClerkImages } = useOptions();
-
-  // TODO: remove this
-  // We don't care about the mutating the reference here
-  const tempStrategyToDisplayData = strategyToDisplayData;
-  const tempProviderToDisplayData = providerToDisplayData;
-  if (!experimental_enableClerkImages) {
-    Object.values(tempStrategyToDisplayData).forEach(strategy => (strategy.iconUrl = svgUrl(strategy.id)));
-    Object.keys(tempProviderToDisplayData).forEach(
-      // @ts-expect-error
-      provider => (tempProviderToDisplayData[provider].iconUrl = svgUrl(provider)),
-    );
-  }
```

Comment:
❓ are we sure about this change?

Reply:
Yes we are! The if was explicitly written so it could be removed when the FF is removed